modules/openapi-generator/src/main/resources/cpp-pistache-server/main-api-server.mustache

```diff
@@ -61,7 +62,8 @@ int main() {
     auto opts = Pistache::Http::Endpoint::options()
         .threads(PISTACHE_SERVER_THREADS);
     opts.flags(Pistache::Tcp::Options::ReuseAddr);
-    opts.maxPayload(PISTACHE_SERVER_MAX_PAYLOAD);
+    opts.maxRequestSize(PISTACHE_SERVER_MAX_REQUEST_SIZE);
+    opts.maxResponseSize(PISTACHE_SERVER_MAX_REQUEST_SIZE);
```

Comment:
I think you meant `PISTACHE_SERVER_MAX_RESPONSE_SIZE` here

Reply:
Thanks for that :D